tests/test_commands.py

```diff
@@ -95,6 +95,43 @@ def test_client_setname(self, r):
         assert r.client_setname('redis_py_test')
         assert r.client_getname() == 'redis_py_test'
 
+    @skip_if_server_version_lt('2.4.0')
+    def test_client_kill(self, r, r3):
+        clients = r.client_list()
+        r.client_kill(clients[1].get('addr'))
+        clients = r.client_list()
+        assert len(clients) == 3
```

Comment:
Same note here as above. This needs to count only connected test runner clients.